utils/variables.js

```diff
@@ -1879,8 +1879,8 @@ const IT_3104 = [
     name: "Atienza, Jansen Karl H.",
     activities: {
       website: {
-        link: "",
-        code: "",
+        link: "https://jansenatienza.github.io/jansen-atienza/",
```

Comment:
both links not working/found

commands in updating branch:
git add .
git commit -m "update links"
git push

```suggestion
        link: "https://jansenatienza.github.io/atienza-jansen/",
```